db/src/test/db.spec.ts

```diff
@@ -31,11 +32,13 @@ test('DB.get returns undefined when no key exists', async (t) => {
   t.is(await db.get('test'), undefined);
 });
 
-test('DB.create creates a new document and returns true', async (t) => {
+test('DB.create creates a new document, returns true, and sets version to 1', async (t) => {
   const { db } = t.context;
+  const t0 = Date.now();
   const ret = await db.create('test', { a: 1 });
-  const val = await db.get('test');
-  t.deepEqual(val, { a: 1 });
+  const doc = await db.getWithMeta('test');
+  t.deepEqual(omit(['updatedAt'], doc), { value: { a: 1 }, version: 1, patches: [] });
```

Comment:
If we injected a `nowFunc` into `db` we could verify it calls `nowFunc` correctly (e.g. it could increase its return value by 13 on each call).

Reply:
I understand but this is good enough IMHO, it gets more complex when it's used for versions (see new changes).